Updates to waste management (#138)
steno-aarhus/research: onboarding/getting-started-plan-4/index.md

If you are to be allocated freezer and refrigeration space, this must be
agreed with Lenette Egelund Pedersen

> ([lenette.pedersen\@clin.au.dk](mailto:lenette.pedersen@clin.au.dk){.email}),

Reviewer: ```suggestion
([lenette.pedersen\@clin.au.dk](mailto:lenette.pedersen@clin.au.dk){.email}).
```

-   Glass waste is placed in cardboard box with blue garbage bag in room
    A401-172.
-   Used batteries are placed in a special container in room A401-172

> -   Mix waste is a waste fraction where you collect 4 different waste for recycling by the following sorting:

Reviewer: ```suggestion
-   Mix waste is a waste fraction where you collect 4 different waste for recycling. Mix waste containers are placed in A401-104 + 106 + 125 + 130 + 172 + Lunchroom. Recycle by the following sorting:
```

       -   *Hard and soft plastic non-hazardous* in room lab A401-104, A401-130 and
        in cell lab A401-122 + A401-123.
        *Hard plastic non-hazardous*, is plastic that has not been in contact with hazardous chemicals or
        is infectious. For example, bottles for media, PBS, sterile
        water, pipette boxes, etc. However, bottles must be rinsed
        before being placed in the waste container and the lid and
        bottle keep apart.
        *Soft plastic*, is plastic such as bubble
        wrap, plastic bags, etc.
     -  *Metal* is for example aluminium foil, metal-metal blister packs and metal cans.

>      -  *Food and Beverage* is for example milk and juice cartons.\

Reviewer: ```suggestion
     -  *Food and Beverage* is for example milk and juice cartons.
```

>        

Reviewer: ```suggestion
```

>     Mix waste containers are placed in A401-104 + 106 + 125 + 130 + 172 + Lunchroom.

Reviewer: ```suggestion
```

>   Please note that food and beverage are not allowed to be brougt and/or consumed in the laboratories eventhough the food/beverage fration picture is presented at the wastebin.   

Reviewer: ```suggestion
 - Please note that food and beverage are not allowed to be brought and/or consumed in the laboratories even though the food/beverage fraction picture is presented at the waste bin.   
```
